src/server/plugins/reactHtml/index.jsx

```diff
@@ -236,6 +240,12 @@ export function getHead({
   `;
 }
 
+export function renderEnvironmentVariables(nonce) {
```

Comment:
just a naming nitpick, `renderEnvironmentVariables` indicates that its a standard practice to include, right now this is okay but could lead to issues in the future